### PR TITLE
Adding metrics to Profiler page

### PR DESCRIPTION
**Pull request by stefantalen**
I was working on a project that did not have the toolbar available in responses (JSON) so I was debugging using the page within the profiler. I found some of the metrics from the toolbar quite useful so hence this PR.

### CODE REVIEW

**Comment by B-Galati** [comment]
@stefantalen Thank you!

I'll look into it asap

**Comment by stefantalen** [comment]
ping @B-Galati

**Comment by B-Galati** [comment]
@stefantalen I think I will be able to check it out in July ;-)

File: Resources/views/Collector/redis.html.twig [hunk 1]
```diff
@@ -102,6 +102,25 @@
...
+                <span class="value">{{ collector.commandcount }}</span>
+                <span class="label">Queries</span>
+            </div>
+
+            <div class="metric">
+                <span class="value">{{ '%0.2f'|format(collector.time) }} <span class="unit">ms</span></span>
+                <span class="label">Query time</span>
+            </div>
+
+            {% if collector.erroredCommandsCount > 0 %}
```
Comment (IonBazan) [inline Resources/views/Collector/redis.html.twig]: I think we should still show **0 Failed Queries**

Comment (stefantalen) [reply Resources/views/Collector/redis.html.twig]: Good suggestion 👍

I've updated the view

**Comment by B-Galati** [comment]
Nice job thanks!

image

**Comment by stefantalen** [comment]
Thanks for merging 👍 , will it be available in the next release (2.1.12)?

**Comment by B-Galati** [comment]
@stefantalen No you'll need to update to the next major version.

The upgrade should not be difficult check our UPGRADE.md file.

**Pull request merged**